### ISSUE TITLE
Update the ungit version

### ISSUE THREAD

**Issue by JanPokorny** [RomanValihura/vscode-ungit-tab#2]
Hi, would it be possible to update the Ungit version used by the plugin? This message shows up every time when opening the Ungit tab:

![obrazek](https://user-images.githubusercontent.com/4580066/42158442-b9c79e14-7df0-11e8-8b83-8ca05bdc3f7d.png)

**Comment by RomanValihura** [comment]
@JanPokorny Yes, it's completely possible. I think that can do it on the weekend.
Thank you for the issue)

**Comment by RomanValihura** [comment]
@JanPokorny Seems like that. So I will compare the extensions and decide what to do with `Ungit tab`

**Comment by JanPokorny** [comment]
@RomanValihura Most notable difference I have found is the lack of a toolbar button (see https://github.com/Hirse/vscode-ungit/issues/54). Also, it has a pipeline and a Greenkeeper set-up to keep Ungit up to date.

**Comment by RomanValihura** [comment]
@JanPokorny mark the plugin as deprecated
